### PR TITLE
feat: UI for displaying profile differences

### PR DESCRIPTION
**Pull request by rajat-mehra05**
#reference-issue: https://github.com/Real-Dev-Squad/identity-service/issues/50

 - This is UI for dashboard site to show profile differences.

### Screenshots:
image

- Mobile Phone responsiveness
image

### CODE REVIEW

File: src/profile/script.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+          designation: newData.designation || '',
+          github_id: newData.github_id || '',
+          linkedin_id: newData.linkedin_id || '',
+          twitter_id: newData.twitter_id || '',
+          instagram_id: newData.instagram_id || '',
+          website: newData.website || '',
+        }),
+      });
+
+      console.log(response);
```
Comment (lakshayman) [inline src/profile/script.js:132]: please remove this

**Review by lakshayman — CHANGES_REQUESTED** [review]

**Comment by lakshayman** [comment]
Can you please remove the button for other users? Only a super user can see it.

**Review by lakshayman — APPROVED** [review]
Looks Good to me

File: src/profile/index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <div id="cover-spin"></div>
+    <nav>
+      <h1>PROFILE DIFFERNCES</h1>
+    </nav>
+
+    <p id="loader">Loading...</p>
+
+    <footer id="footer">
+      <p class="info-repo">
+        The contents of this website are deployed from this
```
Comment (Pavangbhat) [inline src/profile/index.html:21]: image
image

Its good to have a consistent style across different pages of same site

File: src/profile/script.js [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+        method: 'PATCH',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'include',
+        body: JSON.stringify({
+          id: profileDiffId,
+          first_name: newData.first_name || '',
+          last_name: newData.last_name || '',
+          /* email: newData.email || '',
```
Comment (Pavangbhat) [inline src/profile/script.js:119]: Any reason to keep these comments?

File: src/profile/script.js [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+          instagram_id: newData.instagram_id || '',
+          website: newData.website || '',
+        }),
+      });
+
+      if (response.ok) {
+        alert('User Data Approved !!!');
+        window.location.reload();
+      } else {
+        console.log('1');
```
Comment (Pavangbhat) [inline src/profile/script.js:136]: Can we please remove `console.logs`

File: src/profile/script.js [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+  return component;
+}
+
+function wantedData(data) {
+  const {
+    id,
+    first_name,
+    last_name,
+    /*  email,
+    phone, */
```
Comment (Pavangbhat) [inline src/profile/script.js:202]: Any reasons to keep these comments?

File: src/style.css [hunk 6]
```diff
@@ -45,13 +48,38 @@ footer {
...
+  padding: 10px 50px;
+  cursor: pointer;
+  transition: all 0.5s ease;
+}
+.profile-task-btn:hover {
+  color: white;
+  background-color: #1d1283;
+  border: none;
+}
+
```
Comment (Pavangbhat) [inline src/style.css:82]: At some places, we are using **rems** and at some places, we are using **px**
It's a good practice to have consistency

**Review by Pavangbhat — CHANGES_REQUESTED** [review]
Great Work!
I have few suggestions and comments 
Please have a look

File: src/profile/script.js (hunk 5 above)
Comment (rajat-mehra05) [reply src/profile/script.js]: Done

File: src/profile/script.js (hunk 4 above)
Comment (rajat-mehra05) [reply src/profile/script.js]: Done

File: src/profile/script.js (hunk 3 above)
Comment (rajat-mehra05) [reply src/profile/script.js]: Removed.

File: src/profile/index.html (hunk 2 above)
Comment (rajat-mehra05) [reply src/profile/index.html:21]: Fixed.

File: src/style.css (hunk 6 above)
Comment (rajat-mehra05) [reply src/style.css:82]: Fixed.

File: src/profile/script.js (hunk 5 above)
Comment (rajat-mehra05) [reply src/profile/script.js]: Earlier, we were adding those details of users too but then @lakshayman suggested not to add those details for now. Removed the comments for the time being.

File: src/profile/script.js (hunk 4 above)
Comment (rajat-mehra05) [reply src/profile/script.js]: Done

File: src/profile/script.js (hunk 3 above)
Comment (rajat-mehra05) [reply src/profile/script.js]: Earlier, we were adding those details of users too but then @lakshayman suggested not to add those details for now. Removed the comments for the time being.

File: src/profile/index.html (hunk 2 above)
Comment (rajat-mehra05) [reply src/profile/index.html:21]: Fixed.

File: src/profile/script.js (hunk 3 above)
Comment (Pavangbhat) [reply src/profile/script.js]: Thank you

File: src/profile/script.js (hunk 4 above)
Comment (Pavangbhat) [reply src/profile/script.js]: Thank you

**Review by Pavangbhat — APPROVED** [review]
Everything Looks good to me 🎉 
Great work !!

File: src/profile/script.js [hunk 7]
```diff
@@ -0,0 +1,275 @@
+const API_BASE_URL = 'http://localhost:8000';
```
Comment (lakshayman) [inline src/profile/script.js:1]:
```suggestion
const API_BASE_URL = 'https://api.realdevsquad.com';
```

File: src/profile/script.js [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+  dataContainer.appendChild(buttonsContainer);
+
+  const approveBtn = document.createElement('button');
+  const rejectBtn = document.createElement('button');
+  approveBtn.innerText = 'Approve';
+  rejectBtn.innerText = 'Reject';
+
+  approveBtn.onclick = async () => {
+    const reason = prompt('Reason for Approval');
+    if (reason != NULL) {
```
Comment (lakshayman) [inline src/profile/script.js:108]:
```suggestion
    if (reason != null) {
```

File: src/profile/script.js [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+        alert('Something went wrong. Please check console errors.');
+      } finally {
+        document.getElementById('cover-spin').style.display = 'none';
+      }
+    }
+  };
+
+  rejectBtn.onclick = async () => {
+    const reason = prompt('Reason for Rejection');
+    if (reason != NULL) {
```
Comment (lakshayman) [inline src/profile/script.js:149]:
```suggestion
    if (reason != null) {
```

**Review by lakshayman — CHANGES_REQUESTED** [review]

File: src/profile/script.js (hunk 9 above)
Comment (rajat-mehra05) [reply src/profile/script.js:149]: Done.

File: src/profile/script.js (hunk 8 above)
Comment (rajat-mehra05) [reply src/profile/script.js:108]: Done

File: src/profile/script.js (hunk 7 above)
Comment (rajat-mehra05) [reply src/profile/script.js:1]: Done.

**Review by lakshayman — APPROVED** [review]

**Pull request merged**